pawsitive/app/views/services/new.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= form_with model: @service, local: true, class: "max-w-3xl mx-auto p-6 bg-white rounded-lg shadow-md" do |f| %>
+  <h2 class="text-2xl font-bold text-gray-900 mb-4">Create a New Service</h2>
+
+  <!-- Hidden field for member_id -->
+  <%= f.hidden_field :member_id, value: current_user.id %>
+
+  <!-- Service Type Selection -->
+  <div class="mb-4">
+    <%= f.label :service_type, "Service Type", class: "block text-sm font-medium text-gray-700" %>
+    <%= f.select :service_type, options_for_select(['Boarding', 'Walking', 'Day Care']), { prompt: 'Select Service Type' }, class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+  </div>
+
+  <!-- Description Text Area -->
+  <div class="mb-4">
+    <%= f.label :description, "Description", class: "block text-sm font-medium text-gray-700" %>
+    <%= f.text_area :description, class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+  </div>
+
+  <!-- Availability Selection (Checkboxes) -->
+  <div class="mb-4">
+    <%= f.label :availability, "Availability", class: "block text-sm font-medium text-gray-700" %>
+    <% days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'] %>
+    <% days.each do |day| %>
+      <div class="flex items-center mb-2">
+        <%= label_tag "service[availability][#{day}][open]", "#{day} Open", class: "mr-2 text-sm" %>
+        <%= time_field_tag "service[availability][#{day}][open]", '08:00', class: "mr-4 mt-1 block w-1/4 rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
```

Comment:
This is more of a discussion opener on this topic, but would you be open to handling availability at day-of-the-week level?
Rover seems to just use days, and I think that would save us a lot of time in the filters and services page. At least for the MVP.
Here are the examples of how Rover shows that info
![image](https://github.com/chingu-voyages/v49-tier3-team-31/assets/86532479/34b998cb-7a95-4456-97c2-a7bd56433f17)
![image](https://github.com/chingu-voyages/v49-tier3-team-31/assets/86532479/1cb6d4ff-3ede-4523-8fcb-cfd4d59e8aa0)

Reply:
Is it a case you would want to filter by the day as well?